Backend/app/rider-platform/rider-app/Main/src/Domain/Types/HotSpot.hs

```diff
@@ -0,0 +1,59 @@
+module Domain.Types.HotSpot where
+
+import qualified Data.Geohash as DG
+import Data.Text
+import Kernel.External.Maps
+import Kernel.Prelude
+import Kernel.Types.Common hiding (id)
+import Kernel.Utils.Common
+import Tools.Metrics
+
+data HotSpot = HotSpot
+  { geoHash :: Maybe Text,
```

Comment:
let not keep `geoHash` and `centroidLatLong` as Maybe, because if either of them is Nothing, then this hotSpot object is actually not useable and when both are Nothing then for sure not any point, right ?